package.json

```
@@ -21,20 +21,18 @@
 	},
 	"scripts": {
 		"test": "npm-run-all --parallel lint test:unit",
-		"test:unit": "mocha test/test_*.js",
+		"test:unit": "node --test test/test_*.js",
 		"lint": "eslint --cache lib bin/* test test/bin/* && echo ✓"
```

Just to be a bit more explicit (arguably), while we're at it:
```suggestion
		"test:unit": "node --test ./test/test_*.js",
		"lint": "eslint --cache ./lib ./bin/* ./test ./test/bin/* && echo ✓"
```